Resources/public/css/styles.css

```diff
@@ -496,3 +496,15 @@ div.sonata-filters-box div.form-group span.input-group-addon {
     padding: 3px 10px;
     font-size: 13px;
 }
+
+.hide {
+    display: none;
```

Comment:
Either remove the `hide` class and/or add a new option which removes the element completly from the dom.